docs/tans.rst

```diff
@@ -170,3 +170,49 @@ Reference
    :inherited-members:
    :member-order: bysource
    :exclude-members: is_unset, naive_parse, print_nested
+
+
+.. _tans-full-example:
+
+Full example
+------------
+
+A full example on how to get transactions if a TAN is required. If a TAN is required `result` will be an object of type `NeedTANResponse`. Otherwise it will hold your transactions directly.
+
+.. code-block:: python
+
+    import arrow
+    from fints.client import FinTS3PinTanClient, NeedTANResponse
+    from fints.hhd.flicker import terminal_flicker_unix
+
+    client = FinTS3PinTanClient(...)
+    if not client.get_current_tan_mechanism():
+        client.fetch_tan_mechanisms()
+        mechanisms = list(client.get_tan_mechanisms().items())
+        print("Multiple tan mechanisms available. Which one do you prefer?")
+        for i, m in enumerate(mechanisms):
+            print(i, "Function {p.security_function}: {p.name} {p.description_required}".format(p=m[1]))
+        choice = input("Choice: ").strip()
+        client.set_tan_mechanism(mechanisms[int(choice)][0])
```

Comment:
I think it overcomplicates the example to handle all this here, and also it's incomplete (only handles TAN methods, not TAN media which is required for some banks). I'd suggest to just use

```
    from fints.utils import minimal_interactive_cli_bootstrap
    minimal_interactive_cli_bootstrap(f)
```

like the example on the "getting started" page?